modules/test/playwright/tests/portal-workflow-kaleo-designer-web/notification.spec.ts

```diff
@@ -49,9 +52,12 @@ const scriptedRecipientNotification = {
 	templateLanguage: 'text',
 } as Notification;
 
-let workflowDefinitionId: number;
+let workflowDefinitionIds: number[] = [];
+
 let workflowDefinitionName: string;
 
+let scriptManagementEnabled: boolean = false;
```

Comment:
Hey @larissacribeiro, what do you think about use `let scriptManagementDisable boolean = false;`

Reply:
So we just need to handle with this variable in tests that call the `disableScriptManagementConfiguration()` function. The other tests doesn't need any changes about that.

I would like to hear what you think about that 😄

Reply:
We've agreed with your suggestion!